interfaces/builtin/utils_test.go

```diff
@@ -32,19 +32,25 @@ import (
 )
 
 type utilsSuite struct {
-	iface      interfaces.Interface
-	slotOS     *snap.SlotInfo
-	slotApp    *snap.SlotInfo
-	slotSnapd  *snap.SlotInfo
-	slotGadget *snap.SlotInfo
+	iface        interfaces.Interface
+	slotOS       *snap.SlotInfo
+	slotApp      *snap.SlotInfo
+	slotSnapd    *snap.SlotInfo
+	slotGadget   *snap.SlotInfo
+	conSlotOS    *interfaces.ConnectedSlot
+	conSlotSnapd *interfaces.ConnectedSlot
+	conSlotApp   *interfaces.ConnectedSlot
 }
 
 var _ = Suite(&utilsSuite{
-	iface:      &ifacetest.TestInterface{InterfaceName: "iface"},
-	slotOS:     &snap.SlotInfo{Snap: &snap.Info{Type: snap.TypeOS}},
-	slotApp:    &snap.SlotInfo{Snap: &snap.Info{Type: snap.TypeApp}},
-	slotSnapd:  &snap.SlotInfo{Snap: &snap.Info{Type: snap.TypeApp, SuggestedName: "snapd"}},
-	slotGadget: &snap.SlotInfo{Snap: &snap.Info{Type: snap.TypeGadget}},
+	iface:        &ifacetest.TestInterface{InterfaceName: "iface"},
+	slotOS:       &snap.SlotInfo{Snap: &snap.Info{Type: snap.TypeOS}},
+	slotApp:      &snap.SlotInfo{Snap: &snap.Info{Type: snap.TypeApp}},
+	slotSnapd:    &snap.SlotInfo{Snap: &snap.Info{Type: snap.TypeSnapd, SuggestedName: "snapd"}},
```

Comment:
This change is fine although although technically it makes no difference for sanitizeSlotReservedForOS() as at the moment it is based on snap name check. But it's the right thing to do. Thanks.